website/static/js/project-organizer.js

```diff
@@ -372,6 +375,11 @@ var tbOptions = {
                 $('.db-poFilter>input').val('');
             } }, tb.options.removeIcon())
         ];
+
+        if(tb.options.loadValue() < 100){
```

Comment:
I would still like to see this enabled, even if all the projects aren't loaded. Now that there is a loading indicator, is there any reason to keep this disabled?